### PR TITLE
Fix Search user list bug

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

File: spec/lib/users/user_repository_spec.rb [hunk 1]
```diff
@@ -7,19 +7,32 @@ module Users
     let(:http_service) { instance_double('Infrastructure::HttpService') }
     let(:user_repository) { UserRepository.new(http_service) }
     let(:token) { 'sample_token' }
-    let(:last_name) { 'manzo' }
+    let(:params) { { 'lastName' => 'pop' } }
```
Comment (edgibbs) [inline spec/lib/users/user_repository_spec.rb]: can this be new hash style 'lastName': 'pop'

File: spec/lib/users/user_repository_spec.rb [hunk 2]
```diff
@@ -7,19 +7,32 @@ module Users
...
+            .with('/perry/idm/users', params, token)
             .and_return(response)
-          expect(user_repository.get_users(last_name, token)).to eq([])
+          expect(user_repository.get_users(params, token)).to eq([])
+        end
+      end
+
+      context 'with users' do
+        let(:parameter) { { 'lastName' => '' } }
+        it 'without params' do
```
Comment (edgibbs) [inline spec/lib/users/user_repository_spec.rb]: 'returns users' maybe

File: spec/lib/users/user_repository_spec.rb (hunk 1 above)
Comment (edgibbs) [reply spec/lib/users/user_repository_spec.rb]: sounds like no so this is fine.

Comment (RamuKammagani-osi) [reply spec/lib/users/user_repository_spec.rb]: 👍

**Pull request merged**